### PR TITLE
Little changes were made

### PR DESCRIPTION
**Pull request by libertas**
1.Console UI is created.
2.Add a tool for installation.

### CODE REVIEW

File: dict.py [hunk 1]
```diff
@@ -53,6 +55,13 @@ def main():
 				usage()
 			elif opt in ('-w'):
 				dict(arg)
+		if len(sys.argv) == 1:
```
Comment (lcjnil) [inline dict.py:58]: 感觉哪里缩进出了問题嘛。。
话说这么无聊的东西没必要= =

Comment (libertas) [inline dict.py:58]: 0.0 Yah but I was just bored by the homework!!!

**Comment by libertas** [comment]
Merge it !!!

**Comment by lcjnil** [comment]
说啦缩进不对啦！我以前是用tab缩进的

**Pull request closed without merging**